### PR TITLE
Fix #797: Lowfi landscape admin controls

### LINKED ISSUE #797: Lowfi Landscape: Admin Controls
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/e740e866-d87a-4115-9a05-9b4f788dbc92/RG-MN-Register-Empty-30/

Also, add test cases for the same.

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation
Fix #797 Landscape implementation in AdministratorControlsActivity is done.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 1]
```diff
@@ -94,22 +96,38 @@ class AdministratorControlsActivityTest {
 
   @Test
   fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:98]: Please finx indentation in this entire file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, but the indentation in test case file is incorrect. Please fix that. Thanks.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 1 above)
Comment (Luffy18346) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt:98]: resolved.

**Comment by rt4914** [comment]
@Luffy18346 Please don't forget to assign it back to the reviewer.

**Comment by Luffy18346** [comment]
@rt4914 I remember it  but i was waiting for checks to be done

**Comment by rt4914** [comment]
> @rt4914 I remember it but i was waiting for checks to be done

Okay got it. But do not wait for tests to pass. I can understand your situtation but its fine to assign it before test pass. Still its your preference in this case.

**Comment by Luffy18346** [comment]
> > @rt4914 I remember it but i was waiting for checks to be done
> 
> Okay got it. But do not wait for tests to pass. I can understand your situtation but its fine to assign it before test pass. Still its your preference in this case.

Ok

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks

**Pull request merged**